### PR TITLE
added ID type related documentation to Table

### PR DESCRIPTION
**Pull request by alexsb**

### CODE REVIEW

File: src/table/ITable.ts [hunk 1]
```diff
@@ -99,7 +112,12 @@ export interface ITable extends IDataType {
   queryView(name: string, args: IQueryArgs): ITable;
 
   /**
-   * TODO: document
+   * Creates a new view on this table specified by the given range of IDs. A view implements the ITable interface yet is
+   * still backed by the data from the original table.
+   *
+   * When passing a single (1D range) the range applies to the rows.
+   * When passing a 2D range, the row-range is in the first, the col-range in the
```
Comment (sgratzl) [inline src/table/ITable.ts:119]: it doesn't support 2D ranges, since columsn doesn't have unique ids that can be mapped

Comment (thinkh) [reply src/table/ITable.ts:119]: @alexsb Could you please correct it, so that I can merge your PR?

Comment (thinkh) [reply src/table/ITable.ts:119]: @alexsb?

Comment (mstreit) [reply src/table/ITable.ts:119]: @alexsb?

**Comment by thinkh** [comment]
Closed due to no activity.

**Comment by mstreit** [comment]
@alexsb was on vacation until today.

**Comment by thinkh** [comment]
Ok, I didn't know that. If he likes to give input he can restore the branch and re-open the PR.

**Pull request closed without merging**